src/components/tag/adder.js

```diff
@@ -16,6 +16,12 @@ import {
   string
 } from 'prop-types'
 
+export const matchFn = (value, query) => (
```

Comment:
Good idea to add tests!

Let's not export the function separately though, let's just leave it in the default props. The tests can still access the default via `TagAdder.defaultProps.match`